com.unity.render-pipelines.universal/Runtime/UniversalRendererData.cs

```diff
@@ -279,29 +279,7 @@ void ISerializationCallbackReceiver.OnAfterDeserialize()
         {
             if (m_AssetVersion <= 0)
             {
-                var anyNonUrpRendererFeatures = false;
-
-                foreach (var feature in m_RendererFeatures)
-                {
-                    try
-                    {
-                        if (feature.GetType().Assembly == typeof(UniversalRendererData).Assembly)
-                        {
-                            continue;
-                        }
-                    }
-                    catch
-                    {
-                        // If we hit any exceptions while poking around assemblies,
-                        // conservatively assume there was a non URP renderer feature.
-                    }
-
-                    anyNonUrpRendererFeatures = true;
-                }
-
-                // Replicate old intermediate texture behaviour in case of any non-URP renderer features,
-                // where we cannot know if they properly declare needed inputs.
-                m_IntermediateTextureMode = anyNonUrpRendererFeatures ? IntermediateTextureMode.Always : IntermediateTextureMode.Auto;
+                m_IntermediateTextureMode = IntermediateTextureMode.Always;
```

Comment:
You can just remove this whole `if`, since it'll get the default from the field anyway.

Reply:
Addressed in [32f82fb](https://github.com/Unity-Technologies/Graphics/pull/7280/commits/32f82fbb2db56bcc7d77426cf5bbac1e70a06aae)